docs/source/transforms.rst

```diff
@@ -128,3 +128,10 @@ Transforms are common audio transforms. They can be chained together using :clas
 .. autoclass:: Vol
 
   .. automethod:: forward
+  
+:hidden:`SlidingWindowCmn`
```

Comment:
Could you add an equivalent of `sliding_window_cmn_internal` to `functiona.rst`?

Reply:
sure， ths！